src/rust/engine/process_execution/src/remote_tests.rs

```diff
@@ -2409,32 +2371,45 @@ async fn extract_output_files_from_response(
   let executor = task_executor::Executor::new();
   let store_dir = TempDir::new().unwrap();
   let store = make_store(store_dir.path(), &cas, executor.clone());
-  crate::remote::extract_output_files(store, execute_response.get_result(), false)
+  let action_result = execute_response
+    .result
+    .as_ref()
+    .ok_or_else(|| "No ActionResult found".to_string())?;
+  crate::remote::extract_output_files(store, action_result, false)
     .compat()
     .await
 }
 
-pub(crate) fn make_any_proto(message: &dyn Message) -> protobuf::well_known_types::Any {
-  let mut any = protobuf::well_known_types::Any::new();
-  any.set_type_url(format!(
-    "type.googleapis.com/{}",
-    message.descriptor().full_name()
-  ));
-  any.set_value(message.write_to_bytes().expect("Error serializing proto"));
-  any
+pub(crate) fn make_any_proto<T: Message>(message: &T, prefix: &str) -> prost_types::Any {
+  let rust_type_name = type_name::<T>();
+  let proto_type_name = rust_type_name
+    .strip_prefix(prefix)
+    .map(|s| s.clone())
+    .unwrap_or_else(|| rust_type_name.clone())
```

Comment:
I think just `unwrap()`?

Reply:
If the string does not start with the prefix, then `strip_prefix` returns `None`. Probably should be an error if the type name does not start with the passed-in prefix, so `unwrap` would be appropriate.